static/js/components/TopAppBar.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React from "react"
+
+import { routes } from "../lib/urls"
+import MixedLink from "./MixedLink"
+
+import type { CurrentUser } from "../flow/authTypes"
+
+type Props = {
+  currentUser: CurrentUser
+}
+
+const TopAppBar = ({ currentUser }: Props) => (
+  <header className="mdc-top-app-bar">
+    <div className="mdc-top-app-bar__row">
+      <section className="mdc-top-app-bar__section mdc-top-app-bar__section--align-start logo-section">
+        <a href={routes.root}>
+          <img
+            src="/static/images/mitx-pro-logo.png"
+            className="site-logo"
+            alt="MIT xPRO"
+          />
+        </a>
+      </section>
+      <section className="mdc-top-app-bar__section mdc-top-app-bar__section--align-end link-section">
+        {currentUser && currentUser.is_authenticated ? (
```

Comment:
I see this is a carryover from the previous code, but `currentUser` truthy check is not needed as it's always truthy.